spec/controllers/users_controller_spec.rb

```diff
@@ -786,6 +786,20 @@ def post_user
       include_examples 'failed signup'
     end
 
+    context 'with an existing email' do
+      email = 'leo@m.ca'
```

Comment:
I don't completely understand - isn't it local to the context block? - but I've defined it in a `:let` now